addons/tm_bear_fieldmanual_rpg7v2/CfgVehicles.hpp

```diff
@@ -0,0 +1,18 @@
+class CfgVehicles
+{
+	class Man;
+	class CAManBase: Man 
+	{
+		class ACE_SelfActions
+		{
+			class ADDON
+			{
+				displayName = "RPG-7V2 Field Manual";
+				condition = "(secondaryWeapon player) isEqualTo 'rhs_weap_rpg7'";
+				statement = "0 = 0 spawn {findDisplay 46 createDisplay 'RscBear_fieldManual_rpg7v2';};";
```

Comment:
I dont think you need ``0 = 0 spawn..``  but simpley ``findDisplay 46 createDisplay 'RscBear_fieldManual_rpg7v2';``